src/test/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRowMapperTest.java

```diff
@@ -131,11 +131,12 @@ void testSimpleDateFormat_1982_12_31_ERROR() throws Exception {
 		System.out.println(dateTime);
 
 		// ASSERTIONS - these pass if there is a bug in SimpleDateFormat
-		assertEquals("19831231", actual);
-		assertNotEquals("19821231", actual);
+		assertEquals("1982-12-31T12:00", dateTime.toString());
+		assertEquals("19831231", actual, "Java lib error formats as 1983 but should be 1982");
+		assertNotEquals("19821231", actual, "Java lib error formats as 1983 but should be 1982");
 	}
 	@Test
-	void testDateTimeFormat_1982_12_31() throws Exception {
+	void testDateTimeFormat_1982_12_31_ERROR() throws Exception {
```

Comment:
Probably don't need the `throws Exception` on the end of these test methods in this file.

Reply:
maybe not. Just so you know where they come from, I do not add them unless at some point while writing the test there is a checked exception thrown. If the checked exception is removed during iterations then it remains.

I will also remove the println lines too.